samples/basic-nginx/Jenkinsfile

```diff
@@ -0,0 +1,147 @@
+library identifier: "pipeline-library@1.6",
+retriever: modernSCM(
+  [
+    $class: "GitSCMSource",
+    remote: "https://github.com/redhat-cop/pipeline-library.git"
+  ]
+)
+
+
+openshift.withCluster() {
+  env.APP_NAME = "${JOB_NAME}".replaceAll(/-build.*/, '')
+  env.ARTIFACT_DIRECTORY = "samples/basic-nginx/build"
+  env.BUILD = openshift.project()
+  env.DEV = "${APP_NAME}-dev"
+  env.STAGE = "${APP_NAME}-stage"
+  env.PROD = "${APP_NAME}-prod"
+  env.PELORUS = "pelorus"
```

Comment:
@shaheinm as discussed, it may be helpful here to comment that this Jenkinsfile is dynamically finding the route / webhook secret for ease of deployment, but in a more real deployment scenario, the webhook route and webhook secret would likely be defined as env vars / jenkins credentials.